organisations/relationship_transfer.go

```diff
@@ -0,0 +1,94 @@
+package organisations
+
+import (
+	"fmt"
+	"github.com/Financial-Times/neo-utils-go/neoutils"
+	"github.com/jmcvetta/neoism"
+)
+
+type relationships []struct {
+	RelationshipType string `json:"relationship"`
+}
+
+// TransferRelationships is responsible for moving relationships from node with sourceUUID to node with destinationUUID
+func TransferRelationships(cypherRunner neoutils.CypherRunner, destinationUUID string, sourceUUID string) ([]*neoism.CypherQuery, error) {
+
+	relationshipsFromSourceNode, relationshipsToSourceNode, err := getNodeRelationshipNames(cypherRunner, sourceUUID)
+	if err != nil {
+		return nil, err
+	}
+
+	writeQueries := []*neoism.CypherQuery{}
+	for _, rel := range relationshipsFromSourceNode {
+		transfQuery := constructTransferRelationshipsQuery(sourceUUID, destinationUUID, rel.RelationshipType, true)
+		writeQueries = append(writeQueries, transfQuery)
+	}
+
+	for _, rel := range relationshipsToSourceNode {
+		transfQuery := constructTransferRelationshipsQuery(sourceUUID, destinationUUID, rel.RelationshipType, false)
+		writeQueries = append(writeQueries, transfQuery)
+	}
+
+	return writeQueries, nil
+}
+
+func getNodeRelationshipNames(cypherRunner neoutils.CypherRunner, uuid string) (relationshipsFromNodeWithUUID relationships, relationshipsToNodeWithUUID relationships, err error) {
+	// find all the -> relationships
+	relationshipsFromNodeWithUUID = relationships{}
+	readRelationshipsFromNodeWithUUIDQuery := &neoism.CypherQuery{
+		Statement: `match (a:Thing{uuid:{uuid}})-[r]-(b)
+			    where startnode(r) = a
+			    return distinct type(r) as relationship`,
+		Parameters: map[string]interface{}{
+			"uuid": uuid,
+		},
+		Result: &relationshipsFromNodeWithUUID,
+	}
+
+	// find all the <- relationships
+	relationshipsToNodeWithUUID = relationships{}
+	readRelationshipsToNodeWithUUIDQuery := &neoism.CypherQuery{
+		Statement: `match (a:Thing{uuid:{uuid}})-[r]-(b)
+			    where endnode(r) = a
+			    return distinct type(r) as relationship`,
+		Parameters: map[string]interface{}{
+			"uuid": uuid,
+		},
+		Result: &relationshipsToNodeWithUUID,
+	}
+
+	readQueries := []*neoism.CypherQuery{readRelationshipsFromNodeWithUUIDQuery, readRelationshipsToNodeWithUUIDQuery}
+
+	err = cypherRunner.CypherBatch(readQueries)
+
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return relationshipsFromNodeWithUUID, relationshipsToNodeWithUUID, nil
+}
+
+func constructTransferRelationshipsQuery(fromUUID string, toUUID string, predicate string, toRight bool) *neoism.CypherQuery {
```

Comment:
You trying to avoid code duplication makes this method harder to read. Also passing the bool as a param is a code smell IMO. Would it be better to have to methods for transferring inbound/outbound relationships?